build_all/make_sdk_cmds_dict.py

```diff
@@ -0,0 +1,6 @@
+output_path = ''
+
+cmds = {
+    "help":         {'short': 'h', 'text': "Print Help text"},
```

Comment:
>Print [](start = 44, length = 5)

Please make description of command all start with capital or with smaller letter.